### DISCUSSION TITLE
Loops Questions

### DISCUSSION THREAD

**Discussion by CraigButikofer** [USU-ITLS-x265-23-summer/class-repo#157] [General]
If you are a pro at coding loops, please respond!
It is safe to say that loops tend to throw me for a loop. I am struggling to wrap my head around creating loops. Does anyone have any tips or tricks that help them with loops? Any simplified instructional videos that make it a little bit easier to grasp? I spend way too much time on freecodecamp trying to figure it out. I'm sure I'm close, but again they just tend to warp my mind a little.

**Comment by derekscholes** [comment]
Let me know what you think of this video. I think it does a good job at explaining loops and what will be printed in the console and incrementing vs decrementing. 
[JavaScript Loops](https://www.youtube.com/watch?v=Kn06785pkJg)

**Comment by coopsand** [comment]
My biggest tip for loops is using `console.log()` to display the values of the variables you are using in the loop. As someone who has programmed a bit in a couple different programming languages - know that you are not alone in having a difficult time learning loops. I think it is a pretty common thing for people to struggle learning them. I know I did. Another tip is when you start a loop with the value `0`, do not use a conditional that has an "equal to". This will cause your loop to not operate the correct amount of times. If you start a loop with the value `1`, THEN you will use conditionals that have an "equal to". I hope this helped a little. Hang in there!
